src/main/java/org/isf/generaldata/GeneralData.java

```diff
@@ -138,9 +139,9 @@ public final class GeneralData extends ConfigurationProperties {
 	private static final String DEFAULT_PATIENTBILLSTATEMENT = "PatientBillStatement";
 	private static final boolean DEFAULT_DEBUG = false;
 	private static final int DEFAULT_STRONGLENGTH = 10;
-
 	private static final boolean DEFAULT_STRONGPASSWORD = true;
-
+	public static String DEFAULT_PATIENTPHOTO="DB";
```

Comment:
spaces around `=`

Reply:
Thanks, changed also to `private static final`